examples/fastify-typescript-starter/pnpm-workspace.yaml

```
@@ -0,0 +1,4 @@
+packages:
+  - "."
+  - "moose"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)